recipes/install-elasticsearch.rb

```diff
@@ -0,0 +1,57 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: install-elasticsearch
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+
+elk_info = get_elk_info
+
+es_major_version = elk_info[:es_major_version]
+es_version = elk_info[:es_version]
+es_cluster_name = elk_info[:es_cluster_name]
+index_template_path = "#{::Chef::Config[:file_cache_path]}/index-template.json" 
```

Comment:
it feels a little more natural to me to put this somewhere under /etc/ rather than relying on pulling it from what I think of as the chef internals.  Happy to be argued with and proven wrong, though.